src/amo/components/Home/index.js

```diff
@@ -197,72 +194,64 @@ export class HomeBase extends React.Component {
         </Card>
 
         <LandingAddonsCard
-          addons={popularExtensions}
-          className="Home-PopularExtensions"
-          header={i18n.gettext('Popular extensions')}
-          footerText={i18n.gettext('See more popular extensions')}
+          addons={featuredExtensions}
+          className="Home-FeaturedExtensions"
+          header={i18n.gettext('Featured extensions')}
+          footerText={i18n.gettext('See more featured extensions')}
           footerLink={{
             pathname: '/search/',
             query: {
               addonType: ADDON_TYPE_EXTENSION,
-              sort: SEARCH_SORT_POPULAR,
+              featured: true,
             },
           }}
           loading={resultsLoaded === false}
         />
 
         <LandingAddonsCard
-          addons={firstCollection}
+          addons={collections[0]}
           className="Home-FeaturedCollection"
-          header={i18n.gettext('Media downloaders')}
-          footerText={i18n.gettext('See more media downloaders')}
+          header={i18n.gettext('Privacy tools')}
+          footerText={i18n.gettext('See more privacy tools')}
           footerLink={{ pathname:
-            `/collections/${FIRST_COLLECTION_USER}/${FIRST_COLLECTION_SLUG}/`,
+            `/collections/${COLLECTIONS_TO_FETCH[0].user}/${COLLECTIONS_TO_FETCH[0].slug}/`,
           }}
           loading={resultsLoaded === false}
         />
 
         <LandingAddonsCard
-          addons={featuredThemes}
-          className="Home-FeaturedThemes"
-          header={i18n.gettext('Featured themes')}
-          footerText={i18n.gettext('See more featured themes')}
+          addons={popularThemes}
+          className="Home-PopularThemes"
+          header={i18n.gettext('Popular themes')}
+          footerText={i18n.gettext('See more popular themes')}
           footerLink={{
             pathname: '/search/',
             query: {
               addonType: ADDON_TYPE_THEME,
-              featured: true,
+              sort: SEARCH_SORT_POPULAR,
             },
           }}
           loading={resultsLoaded === false}
         />
 
         <LandingAddonsCard
-          addons={topRatedExtensions}
-          className="Home-TopRatedExtensions"
-          header={i18n.gettext('Top-rated Extensions')}
-          footerText={i18n.gettext('See more highly rated Extensions')}
-          footerLink={{
-            pathname: '/search/',
-            query: {
-              addonType: ADDON_TYPE_EXTENSION,
-              sort: SEARCH_SORT_TOP_RATED,
-            },
+          addons={collections[1]}
+          className="Home-FeaturedCollection"
+          header={i18n.gettext('Re-imagine search')}
+          footerText={i18n.gettext('See more search extensions')}
+          footerLink={{ pathname:
+            `/collections/${COLLECTIONS_TO_FETCH[1].user}/${COLLECTIONS_TO_FETCH[1].slug}/`,
```

Comment:
Technically this (and anything else that's just a `pathname`) can be a string instead of an object. So:

```jsx
footerLink={`/collections/${COLLECTIONS_TO_FETCH[1].user}/${COLLECTIONS_TO_FETCH[1].slug}/`}
// instead of:
footerLink={{ pathname:
  `/collections/${COLLECTIONS_TO_FETCH[1].user}/${COLLECTIONS_TO_FETCH[1].slug}/`,
}}
```

Reply:
Thanks, that's nicer.